README.md

```diff
@@ -120,6 +120,7 @@ You can participate by asking or answering open questions in the [issues section
 - [Node.js Official Website](https://nodejs.org/)
 - [Node.js Documentation](https://nodejs.org/docs/latest/api/)
 - [Node.js GitHub Repository](https://github.com/nodejs/node)
-- [Node.js Slack](https://node-js.slack.com/) *([join here](https://www.nodeslackers.com/))*
+- [Node.js Slackers](https://node-js.slack.com/) *([join here](https://www.nodeslackers.com/))*
```

Comment:
the domain is “node slackers” but it’s still the Node.js Slack; i think the name can remain the same

Reply:
It is referred to everywhere as Node Slackers; at the very least, it should be consistent across the board across the other places.

Reply:
(I'm completely fine calling it Node.js Slack, just calling it out that the naming should be consistent across the board)

Reply:
I’d be happy to update everywhere else if you can link me to those places.

Reply:
Or you can search by yourself if this is something that you want to do :) -- I just barely have the time right now with my own tasks. But I believe it would be nodejs/node (a few places there), nodejs/nodejs.org (one place?) and maybe other places.

But calling it Node.js Slack implies it being official maybe? I'd recommend first opening an issue on nodejs/admin to clarify if that is OK (just to be on the safe side 😅)

Reply:
For now, I think using its actual name is better, although this can always change at a later date.

Reply:
It is official. What suggests it’s not?

Reply:
https://nodejs.org/en/about/get-involved#unofficial-discussion-areas suggests that it's an unofficial discussion area.

Reply:
I didn't realize there would a discussion about the name of this Slack link, if you'd like, I can break this into two PRs, one for the Discord (which I assume everyone is fine with), and one for Slack, which might need more discussion.

Reply:
> I clearly missed that in the website redesign; as far as I’m aware irc and slack have been officially endorsed node places for a decade.

In my defense, we did not change the wording. This page was 1:1 copy from the old design :)

Reply:
For now, I've removed the slack related changes, as that's a seperate discussion (which I didn't realize would have to take place).

Reply:
I also see it being mentioned as **unofficial** areas within nodejs/node SUPPORT.md file. So they might as well be unofficial. Can you show any link or document that proves they are official?

Please just open an issue on nodejs/admin goddammit 😅 -- I've asked that a few times in other PRs.

Reply:
Okay. I'm resolving this discussion, as (as @ovflowd mentioned) there may need to be a `nodejs/admin` discussion.